mkdocs.yml

```diff
@@ -67,6 +67,34 @@ nav:
   - examples.md
   - faq.md
   - contributing.md
+  - Future documentation (WIP):
+      - Introduction: working/introduction/introduction.md
+      - Installation:
+          - working/installation/install.md
+          - working/installation/upgrade.md
+      - Getting Started: working/getting_started/getting_started.md
+      - User Guide:
+          - working/user_guide/understanding.md
+          - working/user_guide/creating_testing_collection.md
+          - working/user_guide/testing_existing_collection.md
+          - working/user_guide/testing_existing_roles.md
+          - working/user_guid/ci_integration.md
+          - working/user_guid/using_tox-ansible.md
```

Comment:
What's the theory here with having a user_guide and a user_guid directory?

Reply:
lol, mistake :)